icefoganalytics/travel-authorization
# Switch Use Files To Latest Pattern

Fixes https://github.com/icefoganalytics/travel-authorization/issues/172

# Context

There are currently a bunch of different patterns in place, and its a pain to remember the interface of the of each old pattern.

The latest pattern can be seen in `web/src/use/use-user.js`

# Implementation

Use newer pattern and standardize file naming.
Move `defineExpose` usage to end of setup to handle order of operations scoping.

# Testing Instructions

1. Run the test suite via `dev test` (or `dev test_api`)
2. Boot the app via `dev up`
3. Log in to the app at http://localhost:8080
4. Go to the "My Travel Requests" page via the top drop nav.
5. Open the dev console and check for errors.
6. Create a travel request and go through the various stages while checking the dev console.
